.appveyor.yml

```diff
@@ -1,7 +1,19 @@
 version: 0.5.2.{build}
 os: Windows Server 2012 R2
+environment:
+
+  matrix:
+    # http://www.appveyor.com/docs/installed-software#python
+    - PYTHON: "C:\\Python34"
```

Comment:
Probably? Although I know it's the version installed by default on Amazon Linux. (Aka, what's default on EC2.)